app/views/events/_event.html.erb

```diff
@@ -11,7 +12,8 @@
   <h3>
     <%= event.name %>
     <small>
-      <%= tag_label t('activerecord.attributes.event.draft'), :default if event.draft %>
+      <%= tag_label t('events.notices.is_published'), :success if event.published and can? :view_unpublished, Event %>
+      <%= tag_label t('activerecord.attributes.event.draft'), :default if not event.published and can? :view_unpublished, Event %>
```

Comment:
I think the requirement to show unpublished/published on the view came from the old table layout. In the new layout, just showing a draft label should be enough, since any other event is of course published. @hpi-swt2/product-owner-2016